src/components/utils/useENS.ts

```diff
@@ -0,0 +1,23 @@
+import { ethers } from "ethers";
+import { env } from "../../constants/environmentVariables";
+import { useEffect, useState } from "react";
+
+export function useENS(address: string | null | undefined) {
+  const provider = new ethers.providers.JsonRpcProvider(
+    `https://mainnet.infura.io/v3/${env.INFURA_ID}`
```

Comment:
Make provider network dependant? `(mainnet|testnet).infura.io` ?

Reply:
@xnyl ENS in testnets are not stable and smart contracts are not audited, most of the site uses only in mainnet 🙇🏻  also its available only in rinkeby and ropsten